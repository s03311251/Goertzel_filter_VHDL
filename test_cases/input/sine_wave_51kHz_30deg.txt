2FFF
37E9
3D64
3FE1
3F1F
3B31
347F
2BB7
21BD
1796
0E4A
06CC
01DE
0002
0168
05EA
0D15
162D
2044
2A55
3358
3A63
3EBF
3FF9
3DF1
38DD
3142
27E4
1DB9
13C9
0B17
0486
00C2
002B
02D2
0872
1077
1A11
2446
2E0C
3663
3C73
3F9D
3F8F
3C4B
3625
2DBE
23F1
19BD
102C
0838
02AF
0023
00D5
04B3
0B58
1418
1E0E
2837
318A
3913
3E0F
3FFB
3EA6
3A32
3313
2A03
1FEE
15DB
0CD0
05B9
014F
0005
01FC
0701
0E92
17E9
2213
2C07
34C1
3B5E
3F32
3FD9
3D42
37B0
2FB5
2620
1BEC
1222
09C1
03A4
006A
0068
039C
09B5
1212
1BDB
260F
2FA6
37A4
