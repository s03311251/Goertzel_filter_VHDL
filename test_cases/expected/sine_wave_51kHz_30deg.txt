FFF2CF80
FFEEDF00
